qcodes/instrument_drivers/rohde_schwarz/ZNB.py

```diff
@@ -419,6 +623,79 @@ def _get_sweep_data(self, force_polar: bool = False) -> np.ndarray:
                 self.root_instrument.cont_meas_on()
         return data
 
+    def setup_cw_sweep(self) -> None:
+        """
+        Sets CW mode.
+        """
+
+        # set the channel type to single point msmt
+        self.sweep_type('CW_Point')
+        # turn off average on the VNA since we want single point sweeps.
+        self.write(f'SENS{self._instrument_channel}:AVER:STAT OFF')
```

Comment:
It seems better it this was a boolean paramter. Averaging_enabled or some such thing